__tests__/app.test.js

```diff
@@ -0,0 +1,25 @@
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+const request = require('supertest')
+const app = require('../app/app')
+const db = require('../db/connection')
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe('GET /api/topics', () => {
+    test('should respond with an array of topic objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+            const { topics } = body
+            expect(topics).toHaveLength(3)
+            expect(Array.isArray(topics)).toBe(true)
+            topics.forEach((topic) => {
+                expect(topic).toHaveProperty('description'), expect.any(String)
+                expect(topic).toHaveProperty('slug'), expect.any(String)
+            })
+        })
+    });
+});
```

Comment:
we need to add an error test, what if you went to /api/tooopics instead what error would we get